crates/metadata/lib.rs

```diff
@@ -202,12 +205,16 @@ impl Metadata {
             })
             .unwrap_or(HOST_TARGET);
 
-        // Let people opt-in to only having specific targets
-        let mut targets: HashSet<_> = self
+        let user_targets = self
```

Comment:
Looks quite ambiguous, I first thought that these are coming from cli or similar input, but it seems that it comes from crate's cargo.toml, right? Then might be call it `crate_targets`?

Reply:
Yeah, everything in this file is about Cargo.toml. I'll change the variable name.